src/config/logger.ts

```diff
@@ -77,44 +76,53 @@ export const overrideProbotLoggingMethods = (probotLogger: Logger) => {
 		type: "stream",
 		stream: LOG_STREAM,
 		closeOnExit: false,
-		level: globalLoggingLevel
+		level: defaultLogLevel
```

Comment:
The `globalLoggingLevel` is a `number`? But the `defaultLogLevel` is string "info" by default. Will it be a problem?

Reply:
No.